eng/Versions.props

```
@@ -178,6 +178,14 @@
     <MicrosoftTestPlatformCLIPackageVersion>17.13.0-preview-24569-02</MicrosoftTestPlatformCLIPackageVersion>
     <MicrosoftTestPlatformBuildPackageVersion>17.13.0-preview-24569-02</MicrosoftTestPlatformBuildPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependencies from https://github.com/Microsoft/testfx -->
+    <MicrosoftTestingPlatformPackageVersion>1.7.0-dev</MicrosoftTestingPlatformPackageVersion>
+  </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependencies from https://github.com/Microsoft/testfx -->
+    <MicrosoftTestingExtensionsTrxReportVersion>1.7.0-dev</MicrosoftTestingExtensionsTrxReportVersion>
```

Do we always want to match this with MTP? In that case, maybe:

```suggestion
    <MicrosoftTestingExtensionsTrxReportVersion>$(MicrosoftTestingPlatformPackageVersion)</MicrosoftTestingExtensionsTrxReportVersion>
```